### PR TITLE
FIX: Change build to not update resources for v2.1.1 release

### PR DESCRIPTION
**Pull request by PhotoNomad0**
#### Describe what your pull request addresses (Please include relevant issue numbers):
-

#### Please include detailed Test instructions for your pull request:
-

#### Standard Test Instructions for PR Review Process:

- [ ] Double check unit tests that have been written
- [ ] Check for documentation for code changes
- [ ] Check that there are not inadvertent commits to tC Apps when reviewing a tC Core PR
- [ ] Checkout the branch locally and ensure that app runs as expected
  - [ ] Ensure tests pass
  - [ ] Open and watch the console for errors
  - [ ] Make sure all actions perform as expected
  - [ ] Import and Load a new Project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Switch project to an existing project
  - [ ] Load a tool and perform basic actions
  - [ ] Switch tools and perform basic actions
  - [ ] Next time reverse the order of importing after loading an existing project
- [ ] Reviewer should double check the DoD in the ISSUE, including the “spirit” of the story
- [ ] Ask Ben or Koz if you have any concerns about implementation (especially UI related)

---
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword/translationcore/6704)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,9 @@
     "install-electron-forge": "npm i -g electron-forge@3.2.0",
     "prettier-resource": "prettier --single-quote --print-width 80 --trailing-comma all --write 'tcResources/**/*.json'",
     "doc": "jsdoc -c jsdoc/jsdoc.json",
-    "update-resources": "node scripts/resources/updateResources.js tcResources en hi el-x-koine hbo",
+    "DO-NOT-MERGE-BELOW":"do not merge these changes into develop branch",
```
Comment (mannycolon) [inline package.json]: Is this a script? it looks like only text.

File: package.json [hunk 2]
```diff
@@ -26,7 +26,9 @@
...
+    "update-resources": "# node scripts/resources/updateResources.js tcResources en hi el-x-koine hbo",
+    "TODO":"remove comment (#) above - this is just to disable resource update in v.2.1.1",
```
Comment (mannycolon) [inline package.json]: ibid

**Comment by PhotoNomad0** [comment]
created issue for cleanup: https://github.com/unfoldingWord/translationCore/issues/6706

**Review by PhotoNomad0 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/unfoldingword/translationcore/6704)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @mannycolon)

___
*[package.json, line 29 at r2](https://reviewable.io/reviews/unfoldingword/translationcore/6704#-M0xbvrB-dJQ04lFiM5Q-r2-29:-M0z5r7sExoBy3_oOAEW:b-896fix) ([raw file](https://github.com/unfoldingword/translationcore/blob/edb6d674ced418987c56275b4e1324f2245f21ca/package.json#L29)):*
<details><summary><i>Previously, mannycolon (Manny Colon) wrote…</i></summary><blockquote>

Is this a script? it looks like only text.
</blockquote></details>

Done.
___
*[package.json, line 31 at r2](https://reviewable.io/reviews/unfoldingword/translationcore/6704#-M0xbvrB-dJQ04lFiM5Q-r2-31:-M0z5rnJDpHM7n5TXMdj:b-896fix) ([raw file](https://github.com/unfoldingword/translationcore/blob/edb6d674ced418987c56275b4e1324f2245f21ca/package.json#L31)):*
<details><summary><i>Previously, mannycolon (Manny Colon) wrote…</i></summary><blockquote>

ibid
</blockquote></details>

Done.

**Review by mannycolon — APPROVED** [review]

**Review by mannycolon — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r3.
> *[Reviewable](https://reviewable.io/reviews/unfoldingword/translationcore/6704#-:-M0z8Y6x2U0S7an19ecP:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**